fretboardgtr/fretboard.py

```diff
@@ -29,6 +30,14 @@
     scale_to_enharmonic,
 )
 
+from typing import NewType
+
+# Tuple 
+#  1 : String range of single string
+#  2 : fret 
+#  3 : text (e.g. finger number)
+FingerPos = NewType('FingerPos', Tuple[Union[ Tuple[int,int], int], int, Optional[str]]) 
```

Comment:
It's a personal preference but in your issue example #31 I found it pretty hard to read : 

```
g_maj = [
    ((0,5), 3, "1"),
    (1, 5 , "4"),
    (2, 5 , "3"),
    (3, 4 , "2")
] 
```

It is not obvious what is what, but when you know it it's pretty easy. Maybe we just need documentation or mabe another data structure like 

```
FingerPosition(string_position=1, fret=3, text="1")
FingerPosition(string_interval=[0,5}, fret=3, text="1")
FingerPosition(string_interval=[0,5}, fret=3, finger_number=1)
```

Or maybe 
```
FingerPosition(position=1, fret=3, finger_number=1)
FingerBarre(from_string=0, to_string=5, fret=3, finger_number=1)
```

Then your type could be 
```
FingerSet = List[FingerPosition, FingerBarre]
```

What do you think ?